server/db/clean-database-backup.sql

```diff
@@ -0,0 +1,27 @@
+-- sql for purging pools and its' related entities.
+-- helpful for database backups where ExpiredPoolsJob has not run for a long time.
+SET FOREIGN_KEY_CHECKS=0;
+
+DELETE FROM cp_pool_attribute WHERE pool_id = (SELECT id FROM cp_pool WHERE enddate < now() AND id = cp_pool_attribute.pool_id);
```

Comment:
If we're just using this for internal testing, it should be fine. With the formal review, I thought this was intended for wider audiences.

Since it'll be part of our codebase going forward, we should probably move it to server/fixes and add a comment or two indicating that it doesn't respect certain edge cases with regards to end dates.